src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -99,22 +40,13 @@ function GridContainer(props: RelayProps) {
       relay.loadMore(PAGE_SIZE, x => x)
     })
 
-  const handleScroll = ({ nativeEvent }) => {
-    if (isCloseToBottom(nativeEvent)) {
-      loadMore()
-    }
-  }
-
   return (
-    <ScrollView onScroll={handleScroll} scrollEventThrottle={30}>
+    <ScrollView onScroll={isCloseToBottom(loadMore)} scrollEventThrottle={400}>
       <SectionHeader title={title} />
       <Container>
         <GenericGrid artworks={artworks} />
+        {relay.isLoading() && <Spinner style={{ marginTop: 20 }} />}
```

Comment:
In the interests of deadlines I went with this approach rather than trying to refactor <InfiniteScrollArtworkGrid /> to handle content that isn't artist-generic and introduce further complexity. Can return at a later point

Reply:
👍